content/ADR-125-realm-modifiers-for-worlds-in-renderer.md

```diff
@@ -0,0 +1,191 @@
+When needed, this modifier will implement the change of the minimap image; as a user generated world will have the possibility to show its own minimap png.
+	
+
+### Data Structure 
+
+`AboutResponse` contains the info of the realm which has just been accessed. This class is autogenerated according to protobuff specifications. Each modifier gets access to the AboutResponse, and gets the info necessary to execute its behaviour. It has the following structure: 
+
+```typescript
+type About = {
+  healthy: boolean
+  configurations: {
+    networkId: number
+    realmName?: string
+    scenesUrn: string[]
+    globalScenesUrn: string[]
+  }
+  content: {
+    healthy: boolean
+    version?: string
+    commitHash?: string
+    publicUrl: string
+  }
+  comms: {
+    healthy: boolean
+    protocol: string
+    fixedAdapter?: string
+    usersCount?: number
+  }
+  lambdas: {
+    healthy: boolean
+    version?: string
+    commitHash?: string
+    publicUrl: string
+  }
+  bff?: {
+    healthy: boolean
+    userCount: number
+    commitHash?: string
+    publicUrl: string
+  }
+}
+```
+
+Example:
+
+```json
+{
+  "healthy": true,
+  "content": {
+    "healthy": true,
+    "version": "5.1.2",
+    "commitHash": "dce61c002c89db4966c0cdd008d3d654f297050b",
+    "publicUrl": "https://peer.decentraland.org/content"
+  },
+  "configurations": {
+    "realmName": "catalyst-4",
+    "networkId": 1
+  },
+  "comms": {
+    "healthy": true,
+    "protocol": "v3",
+    "fixedAdapter": "ws-room:mini-comms.decentraland.org/rooms/test-room"
+  },
+  "lambdas": {
+    "healthy": true,
+    "version": "5.1.2",
+    "commitHash": "dce61c002c89db4966c0cdd008d3d654f297050b",
+    "publicUrl": "https://peer.decentraland.org/lambdas"
+  },
+  "bff": {
+    "healthy": true,
+    "commitHash": "369c5dafeda62a1b16f5232cd477565cc3f3d513",
+    "userCount": 0,
+    "publicUrl": "/"
+  }
+}
+```
+
+## Current Implementation 
+
+![Untitled Diagram drawio](https://user-images.githubusercontent.com/1999557/202185605-65ebf318-a2bd-40c9-8908-200daa2166b5.png)
+
+
+## Task breakdown 
+
+To complete the Beta version for worlds, the following modifiers are required:
+
+- **Skybox Modifier (V1 task):** Create a skybox modifier to allow the world creator control its state.
+	- Skybox settings (time and dynamic toggle) will be unavaible while in world; they should be non-interactable.
+	- A warning message should be added in settings to explicit to the user that the skybox is being controller by the creator
+	- A specific time should be added to the AboutResponse.Configuration to allow the creators set the time of their World
+	- Skybox should be set accordingly to the new variable mentioned above
+
+- **Green Blockers Modifiers:** Extra functionality should be added to the current green blockers modifier.
```

Comment:
Should we use the same naming for consistency and replace `green blockers modifier` to  `Realms Blocker Modifier`

Reply:
Done!